examples/field_formats_example/public/examples/2_creating_custom_formatter.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { KBN_FIELD_TYPES } from '@kbn/field-types';
+import { FieldFormat, SerializedFieldFormat } from '../../../../src/plugins/field_formats/common';
+import { FieldFormatsSetup, FieldFormatsStart } from '../../../../src/plugins/field_formats/public';
+
+// 1. Create a custom formatter by extending {@link FieldFormat}
+export class ExampleCurrencyFormat extends FieldFormat {
+  static id = 'example-currency';
+  static title = 'Currency (example)';
+
+  // 2. Specify field types that this formatter supports
+  static fieldType = KBN_FIELD_TYPES.NUMBER;
```

Comment:
What if the formatter supports more than one type?

Reply:
Clarified 👍